packages/react-examples/.storybook/preview.js

```diff
@@ -244,23 +237,6 @@ function generateStoriesFromExamples(key, stories, req) {
     // @ts-ignore -- PACKAGE_NAME is replaced by a loader
     const isReactPackageStory = 'PACKAGE_NAME' === 'react';
 
-    // @TODO
```

Comment:
If all the collocated stories have authoritative titles, we do not need any magic here.

Reply:
right but this has another restriction that we need to provide that package suffix within collocated stories -> I provided additional thoughts on that in other comment.

For now I think this is good enough change with additional tax to provide those suffixes per collocated story.